Makefile

```diff
@@ -49,6 +49,9 @@ _test:
 
 _install:
 	$(JSONNETBUNDLERCMD) install
+	# Hack kubernetes-mixin to FC Fork until https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/517 is merged
```

Comment:
Could we fork this to mintel repo instead?

Reply:
I am expecting this to be Merged soon, so i will just revert to upstream and keep the issue we had anyway since day 1